src/BaGetter.Azure/Storage/BlobStorageService.cs

```diff
@@ -65,7 +65,7 @@ public Task<Uri> GetDownloadUriAsync(string path, CancellationToken cancellation
             }
             catch (StorageException e) when (e.IsAlreadyExistsException())
             {
-                using (var targetStream = await blob.OpenReadAsync(cancellationToken))
+                using (var targetStream = await blob.OpenReadAsync())
```

Comment:
No token either?

Reply:
The api changed: They discarded the CT as single argument /facepalm